vuepress/docs/v6.3.2/tutorials/samples/hub.md

```diff
@@ -51,33 +51,33 @@ TODO: screenshot of the 6.3.2 AppBuilder with both bundles available but not ins
 (Entando 6.3.2) There is a cache issue when deploying the Hub bundles where not all widgets or MFEs initially appear on some pages, particularly the Dashboard page. Restarting the quickstart-server pod (which holds the Entando App Engine) will clear the cache. This is only necessary on the initial install.
 :::
 
-5. The Hub can be accessed from the App Builder by navigating to `Pages → Management`, finding `xxxx` in the page tree, and clicking `View Published Page` from its actions.
+5. The Hub can be accessed from the App Builder by navigating to `Pages → Management`, finding `Hub Test` **change** in the page tree, and clicking `View Published Page` from its actions.
```

Comment:
According to the above instructions, you're in the App Builder already, so thinking you can shorten the beginning because the sentence is dense. Not a big deal, it's understandable as it is.